webrender_api/src/display_list.rs

```diff
@@ -494,6 +495,23 @@ impl DisplayListBuilder {
         self.push_item(item, rect, local_clip);
     }
 
+    pub fn push_line(&mut self,
+                     local_clip: Option<LocalClip>,
+                     baseline: f32,
+                     start: f32,
```

Comment:
All our rectangles are exclusive on one end, in a way: https://github.com/servo/euclid/blob/master/src/rect.rs#L180
So if you are using the start/end to generate a rectangle, it should play along just fine.

But really, that exclusiveness deserves some dedicated investigation, so I'm totally fine without `Range` here